packages/forms/docs/stories/validators/validators.story.js

```diff
@@ -0,0 +1,197 @@
+                />
+            </UniversalForm>
+            <StyledButton label="Submit" type="submit" form="isUrl" variant="brand" />
+        </Container>
+    </RainbowFirebaseApp>
+);
+
+export const Max = () => (
+    <RainbowFirebaseApp>
+        <Container>
+            <UniversalForm id="max" onSubmit={onSubmit}>
+                <Field
+                    name="age"
+                    component={Input}
+                    label="Age"
+                    placeholder="Enter your age"
+                    labelAlignment="left"
+                    validate={max(120, 'Humans can not live more than 120 years')}
+                    className="rainbow-m-bottom_large"
+                    type="number"
+                />
+            </UniversalForm>
+            <StyledButton label="Submit" type="submit" form="max" variant="brand" />
+        </Container>
+    </RainbowFirebaseApp>
+);
+
+export const Min = () => (
+    <RainbowFirebaseApp>
+        <Container>
+            <UniversalForm id="min" onSubmit={onSubmit}>
+                <Field
+                    name="age"
+                    component={Input}
+                    label="Age"
+                    placeholder="Enter your age"
+                    labelAlignment="left"
+                    validate={min(0, 'The value must be a positive number')}
+                    className="rainbow-m-bottom_large"
+                    type="number"
+                />
+            </UniversalForm>
+            <StyledButton label="Submit" type="submit" form="min" variant="brand" />
+        </Container>
+    </RainbowFirebaseApp>
+);
+
+export const MaxLength = () => (
+    <RainbowFirebaseApp>
+        <Container>
+            <UniversalForm id="maxLength" onSubmit={onSubmit}>
+                <Field
+                    name="bio"
+                    component={Textarea}
+                    label="Bio"
+                    placeholder="Describe yourself"
+                    labelAlignment="left"
+                    validate={maxLength(150)}
+                    className="rainbow-m-bottom_large"
+                />
+            </UniversalForm>
+            <StyledButton label="Submit" type="submit" form="maxLength" variant="brand" />
+        </Container>
+    </RainbowFirebaseApp>
+);
+
+export const MinLength = () => (
+    <RainbowFirebaseApp>
+        <Container>
+            <UniversalForm id="minLength" onSubmit={onSubmit}>
+                <Field
+                    name="bio"
+                    component={Textarea}
+                    label="Bio"
+                    placeholder="Describe yourself"
+                    labelAlignment="left"
+                    validate={minLength(50)}
+                    className="rainbow-m-bottom_large"
+                />
+            </UniversalForm>
+            <StyledButton label="Submit" type="submit" form="minLength" variant="brand" />
+        </Container>
+    </RainbowFirebaseApp>
+);
+
```

Comment:
here we should also add examples using composeValidators and using more than one validator like for example when you want to validate an email, first you want to check if it required and then you want to check if it is a valid email